### PR TITLE
Create trench map markers

### PR DESCRIPTION
**Pull request by Seb105**
- Create map marker for placer side & all friendly sides on trench creation
- JIP compatible
- Remove on trench removal

### CODE REVIEW

**Comment by Dahlgren** [comment]
Would be nice to have configurable via CBA Settings

**Comment by Seb105** [comment]
Added CBA setting, someone else will have to do the german stringtable 😂

File: addons/main/script_version.hpp [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #define MAJOR 1
 #define MINOR 6
-#define PATCHLVL 8
+#define PATCHLVL 9
```
Comment (AdlerSalbei) [inline addons/main/script_version.hpp:3]: Please refrain from changing the version.

File: addons/functions/functions/fnc_setMarkerVisible.sqf [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ *
+ * Example:
+ * _trench remoteExec ["grad_trenches_functions_fnc_setMarkerVisible", 0, _trench];
+ *
+ * Public: No
+ */
+
+
+// Spawn necessary for JIP, as you must wait until player object exists to determine its side.
+_this spawn {
```
Comment (AdlerSalbei) [inline addons/functions/functions/fnc_setMarkerVisible.sqf:19]: Use the cba_fnc_waitUntilAndExecute instead of spawn + waitUntil

**Pull request merged**